### ISSUE TITLE
Json Date Malformed

### ISSUE THREAD

**Issue by markux** [The-Golden-At-SRL/CatLendar#2]
[At this point](https://github.com/The-Golden-At-SRL/CatLendar/blob/d96338938ed6e07c03554939d064cc285e22466d/CatLendar.py#L147)

the response is in this manner:
```
```json\n{\n    \"date\": \"10/05/2024\",\n    \"time\": \"15:30\"\n}\n```
```

so with this change it works:
```
dateJson = json.loads(response.replace("```json", "").replace("```", ""))
```

**Comment by jacopopalumbo01** [comment]
@markux thanks for the issue report! It is now solved with PR #3

**Comment by jacopopalumbo01** [comment]
@markux thanks for the issue report! It is now solved with PR #3
